game/src/main/java/org/apollo/game/model/entity/Player.java

```diff
@@ -1035,4 +1040,30 @@ private void initSkills() {
 		skillSet.addListener(new LevelUpSkillListener(this));
 	}
 
+	/**
+	 * Decrement run energy.
+	 *
+	 * @param weight the amount of weight to be taken into account when
+	 * determining how much run energy will be depleted.
+	 */
+	public void decrementRunEnergy(double weight) {
```

Comment:
@Major- any thoughts on this being in core? Seems like it might be the responsibility of a plugin, but I'm not sure.

Reply:
I think core is correct